src/main/java/org/opentripplanner/updater/GraphUpdaterManager.java

```diff
@@ -78,14 +78,15 @@ public GraphUpdaterManager(Graph graph, TransitModel transitModel, List<GraphUpd
     this.graph = graph;
     this.transitModel = transitModel;
     // Thread factory used to create new threads, giving them more human-readable names.
-    var threadFactory = new ThreadFactoryBuilder().setNameFormat("updater-%d").build();
-    this.scheduler = Executors.newSingleThreadScheduledExecutor(threadFactory);
+    var graphWriterThreadFactory = new ThreadFactoryBuilder().setNameFormat("graph-writer").build();
```

Comment:
```suggestion
    var graphWriterThreadFactory = new ThreadFactoryBuilder().setNameFormat("graph-writer-%d").build();
```

Reply:
There is only one thread, the counter does not bring additional information.